site/docs/components/badge/usage.mdx

```diff
@@ -0,0 +1,38 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  label: Usage
+data:
+  $ref: ./#/data
+---
+
+### Using the Badge component
+
+#### When not to use Component Name
+
+- Badge must not indicate status through color, such as red-amber-green or progress. Instead, use the Pill component.
+- Don’t allow users to edit the contents of Badge. To accept user input, consider using an interactive Pill.
+
+### Import
+
+{/* Update the text and code snippet below as needed: */}
+
+To import Badge from the core Salt package, use:
+
+```js
+import { Badge } from "@salt-ds/core";
+```
+
+### Props
+
+{/* TODO: Update this to grab from core package when component is released */}
+
+<PropsTable packageName="labs" componentName="Badge" />
```

Comment:
```suggestion
<PropsTable packageName="core" componentName="Badge" />
```
might be a good idea to have it as packageName="core" (in alignment with what's mentioned above in Import section, and all other component docs)